constants.py

```diff
@@ -605,3 +605,22 @@
 kPhotonvisionCameraPixelDimensions = (960, 720)
 kApriltagWidth = 8.125 * kMetersPerInch
 kApriltagHeight = kApriltagWidth
+
+# CubeCone Motors
+kConeCubeCANID = 3
+kConeCubePIDSlot = 0
+kConeCubePGain = 0.0001
+kConeCubeIGain = 0
+kConeCubeDGain = 0
+kIntakeMotorPercent = 0.7
+"""revolutions per minute (rpm)"""
+
+kCubeLoaded = "cubeInPos"
```

Comment:
this should be noted that it's a key
`kCubeLoadedKey` and `kConeLoadedKey`

for all smartdashboard key stuff do note it in variable naming